packages/hash/frontend/src/pages/[account-slug]/entity-types/property-list-card.tsx

```diff
@@ -0,0 +1,701 @@
+            },
+            [`.${tableCellClasses.body} .${checkboxClasses.root}`]: {
+              textAlign: "center",
+            },
+            [`.${tableBodyClasses.root} .${tableRowClasses.root}`]: {
+              [`.${tableCellClasses.root}`]: {
+                "&:first-child": {
+                  borderTopLeftRadius: 1,
+                  borderBottomLeftRadius: 1,
+                },
+                "&:last-child": {
+                  borderTopRightRadius: 1,
+                  borderBottomRightRadius: 1,
+                },
+              },
+              [`&:hover .${tableCellClasses.root}`]: {
+                background: theme.palette.gray[10],
+              },
+            },
+          })}
+        >
+          <TableHead>
+            <Typography
+              component={TableRow}
+              variant="smallTextLabels"
+              sx={{
+                fontWeight: 600,
+              }}
+            >
+              <TableCell>Property name</TableCell>
+              <TableCell sx={{ width: 180 }}>Expected values</TableCell>
+              <TableCell
+                sx={{
+                  width: "170px !important",
+                }}
+              >
+                Allow multiple values <QuestionIcon />
+              </TableCell>
+              <TableCell sx={{ width: 100 }}>Required</TableCell>
+              <TableCell sx={{ width: 150 }}>
+                Default value <QuestionIcon />
+              </TableCell>
+              <TableCell sx={{ width: 70 }} />
+            </Typography>
+          </TableHead>
+          <TableBody>
+            {created.map((id) => (
+              <TableRow key={id}>
+                <TableCell>
+                  <Typography variant="smallTextLabels" fontWeight={500}>
+                    Share Price
+                  </Typography>
+                </TableCell>
+                <TableCell>
+                  <Chip label="Number" />
+                </TableCell>
+                <TableCell sx={{ textAlign: "center" }}>
+                  <Checkbox />
+                </TableCell>
+                <TableCell sx={{ textAlign: "center" }}>
+                  <Checkbox />
+                </TableCell>
+                <TableCell sx={{ px: "0px !important" }}>
+                  <TextField
+                    placeholder="Add default value"
+                    sx={{
+                      width: "100%",
+                      [`.${tableRowClasses.root}:not(:hover) & .${outlinedInputClasses.root}:not(:focus-within)`]:
+                        {
+                          boxShadow: "none",
+                          [`.${outlinedInputClasses.notchedOutline}`]: {
+                            borderColor: "transparent",
+                          },
+                          [`.${outlinedInputClasses.input}::placeholder`]: {
+                            color: "transparent",
+                          },
+                        },
+                    }}
```

Comment:
What do you think of adding a transition here?

```suggestion
                   sx={({ transitions }) => ({
                      width: "100%",
                      [`.${outlinedInputClasses.notchedOutline}`]: {
                        transition: transitions.create("border-color"),
                      },
                      [`.${outlinedInputClasses.input}::placeholder`]: {
                        transition: transitions.create("color"),
                      },
                      [`.${tableRowClasses.root}:not(:hover) & .${outlinedInputClasses.root}:not(:focus-within)`]:
                        {
                          boxShadow: "none",
                          [`.${outlinedInputClasses.notchedOutline}`]: {
                            borderColor: "transparent",
                          },
                          [`.${outlinedInputClasses.input}::placeholder`]: {
                            color: "transparent",
                          },
                        },
                    })}
```